src/variables/modifiers.md

```diff
@@ -0,0 +1,35 @@
+## Modifiers
+### Intro: 
+
+| Modifier | Description |
+| --- | --- |
+| public | Accessible from any other class |
+| private | Accessible only within the class they are declared |
+| protected | Accessible within the package and outside the package but through inheritance only |
+| static | Belongs to the class rather than the object |
+| final | Value can't be changed |
+| abstract | Can't be instantiated |
+| synchronized | Threads handle the method or block of the method sequentially |
+| volatile | Value can be modified by different threads |
+| default | No modifier specified, accessible within the same package |
```

Comment:
There is actually a `default` keyword for interfaces so thats a little confusing